providers/gcp/instances.go

```diff
@@ -37,7 +37,7 @@ func (g InstancesGenerator) createResources(ctx context.Context, instancesList *
 	resources := []terraformutils.Resource{}
 	if err := instancesList.Pages(ctx, func(page *compute.InstanceList) error {
 		for _, obj := range page.Items {
-			if strings.HasPrefix("gke-", obj.Name) {
+			if strings.HasPrefix(obj.Name, "gke-") {
```

Comment:
nice catch :)

Reply:
As per https://github.com/reviewdog/action-golangci-lint#filter_mode, `reviewdog/action-golangci-lint` was only looking at the code that were `Added `, hence all these errors were not reported.

I think it makes sense to have it that way.